.github/workflows/docs.yml

```diff
@@ -4,13 +4,13 @@ on:
   push:
     branches:
       - main
-  pull_request:
```

Comment:
if we want to test the docs, we'll need a secondary deployment. I don't like the docs being regenerated on pull requests as that leads us to having broken docs